processing/src/main/java/org/apache/druid/segment/serde/DictionaryEncodedStringIndexSupplier.java

```diff
@@ -280,8 +351,10 @@ public ImmutableBitmap next()
 
             private void findNext()
             {
-              while (next < 0 && iterator.hasNext()) {
-                ByteBuffer nextValue = iterator.next();
+              // if the size of in-filter values is less than the threshold percentage of dictionary size, then use binary search
+              // based lookup per value. The algorithm works well for smaller number of values.
```

Comment:
I *think* this comment is a bit stale now as the choice of which algorithm to use happens earlier in the code than this line.

Reply:
thanks for the catch! :+1: moved up the comment to the correct location